wftests/warrior_tests/data/data_wtag.xml

```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" ?>
```

Comment:
Move this data_file to framework_tests directory since the tc is placed inside framework_tests.

Reply:
Completed.